example/README.md

```diff
@@ -155,6 +155,17 @@ Run `terraform apply`.
 
 This will deploy your data-migration resources.
 
+## Enable a distribution API
+
+The steps below assume you will be using the [Cumulus Distribution API](./cumulus-tf/cumulus_distribution.tf) and no additional changes are required unless you want to use the Thin Egress App instead. If you would prefer
+to use the Thin Egress App (TEA), uncomment the TEA-specific variables in the "cumulus" module in [example/cumulus-tf/main.tf](./cumulus-tf/main.tf).
+
+**Note:** If you choose to deploy TEA by making the above change in [example/cumulus-tf/main.tf](./cumulus-tf/main.tf), both TEA and the Cumulus Distribution API
+will be deployed but only TEA will be enabled for use. If you wish to use TEA and don't want to deploy both, comment or delete the following:
```

Comment:
```suggestion
**Note:** Both TEA and the Cumulus Distribution API are deployed by default but if you make the above change in [example/cumulus-tf/main.tf](./cumulus-tf/main.tf), only TEA will be enabled for use. If you wish to use TEA and don't want to deploy both, comment or delete the following:
```

Nitpicking, but this clarified it for me. The key is that you're not deploying TEA when making the above change because it is still deployed by default, you're just making it functional